docs/materials/Shadows.stories.mdx

```diff
@@ -17,20 +17,43 @@ All Tachyons shadows utilities have been replaced to match Orbit Design values.
 <Table
     columns={[
         { title: "Class", headerStyle: { width: "200px" } },
-        { title: "Variable", headerStyle: { width: "225px" } },
-        { title: "", headerStyle: { width: "400px" } }
-    ]}
+        { title: "Variable", headerStyle: { width: "300px" } },
+        { title: "Sample", headerStyle: { width: "350px" } } ]}
     rows={[
-        [{ value: "shadow-100", className: "code f7 o-90" }, { value: "--o-ui-global-shadow-100", className: "code f7 o-90" }, <div className="shadow-100 h9 ma3"></div>],
-        [{ value: "shadow-200", className: "code f7 o-90" }, { value: "--o-ui-global-shadow-200", className: "code f7 o-90" }, <div className="shadow-200 h9 ma3"></div>],
-        [{ value: "hover-shadow-100", className: "code f7 o-90" }, { value: "--o-ui-global-shadow-100", className: "code f7 o-90" }, <div className="hover-shadow-100 ba border-1 h9 ma3"></div>],
-        [{ value: "hover-shadow-200", className: "code f7 o-90" }, { value: "--o-ui-global-shadow-200", className: "code f7 o-90" }, <div className="hover-shadow-200 ba border-1 h9 ma3"></div>],
-        [{ value: "focus-shadow-100", className: "code f7 o-90" }, { value: "--o-ui-global-shadow-100", className: "code f7 o-90" }, <div className="focus-shadow-100 ba border-1 h9 ma3 outline-0" tabIndex="0"></div>],
-        [{ value: "focus-shadow-200", className: "code f7 o-90" }, { value: "--o-ui-global-shadow-200", className: "code f7 o-90" }, <div className="focus-shadow-200 ba border-1 h9 ma3 outline-0" tabIndex="0"></div>]
+        [{ value: "shadow-100", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-100", className: "code f8 o-90" }, <div className="shadow-100 br2 bg-white h9 ma3"></div>],
+        [{ value: "shadow-200", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-200", className: "code f8 o-90" }, <div className="shadow-200 br2 bg-white h9 ma3"></div>],
+        [{ value: "shadow-300", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-300", className: "code f8 o-90" }, <div className="shadow-300 br2 bg-white h9 ma3"></div>],
+        [{ value: "shadow-400", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-400", className: "code f8 o-90" }, <div className="shadow-400 br2 bg-white h9 ma3"></div>]
     ]}
     rowClassName="background-1"
 />
 
+CSS shadows are made using a system of 6 layers in order to provide silky smooth shadows, the values below can be used [here](https://shadows.brumm.af/) in order to preview and play with the shadows.
```

Comment:
Would change 

"CSS shadows are made using a system of 6 layers in order to provide silky smooth shadows, the values below can be used [here](https://shadows.brumm.af/) in order to preview and play with the shadows."

For:

CSS shadows are made using a system of 6 layers in order to provide silky smooth shadows. The shadows values can be previewed and altered on [shadows.brumm.af](https://shadows.brumm.af/).